corehq/motech/dhis2/models.py

```diff
@@ -32,6 +35,19 @@ class Dhis2Connection(models.Model):
     password = models.CharField(max_length=255, null=True)
     skip_cert_verify = models.BooleanField(default=False)
 
+    @property
+    def plaintext_password(self):
+        pass  # TODO: ...
+
+    @plaintext_password.setter
+    def plaintext_password(self, plaintext):
+        # Use simple symmetric encryption. We don't need it to be
+        # strong, considering we'd have to store the algorithm and the
+        # key together anyway; it just shouldn't be plaintext.
+        # (2020-03-09) Not true. The key is stored separately.
```

Comment:
Once these passwords are migrated to more robust encryption, `Dhis2Connection` instances will be deleted.